src/Product/Receipt/ReceiptV5LineItems.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Mindee\Product\Receipt;
+
+/**
+ * List of line item details.
+ */
+class ReceiptV5LineItems extends \ArrayObject
+{
+    /**
+     * @param array        $rawPrediction Raw prediction array.
+     * @param integer|null $pageId        Page number for multi pages document.
+     */
+    public function __construct(array $rawPrediction, ?int $pageId = null)
+    {
+        $entries = [];
+        foreach ($rawPrediction as $entry) {
+            $entries[] = new ReceiptV5LineItem($entry, $pageId);
+        }
+        parent::__construct($entries);
+    }
+
+    /**
+     * Creates a line of rST table-compliant string separators.
+     *
+     * @param string $char Character to use as a separator.
+     * @return string
+     */
+    public static function lineItemsSeparator(string $char): string
+    {
+        $outStr = "  ";
+        $outStr .= "+" . str_repeat($char, 38);
+        $outStr .= "+" . str_repeat($char, 10);
+        $outStr .= "+" . str_repeat($char, 14);
+        $outStr .= "+" . str_repeat($char, 12);
+        return $outStr . "+";
+    }
+
+
+    /**
+     * String representation.
+     *
+     * @return string
+     */
+    public function lineItemsToStr(): string
```

Comment:
```suggestion
    public function __toString(): string
```

Reply:
Oh yeah that makes more sense :sweat_smile: